python.d/go_expvar.chart.py

```diff
@@ -89,9 +78,32 @@ def __init__(self, configuration=None, name=None):
             self.order = list()
 
         # if extra charts are defined, parse their config
+        extra_charts = self.configuration.get('extra_charts', dict())
         if len(extra_charts) > 0:
             self._parse_extra_charts_config(extra_charts)
 
+    def check(self):
+        """
+        Check if the module can collect data:
+        1) At least one JOB configuration has to be specified
+        2) The JOB configuration needs to define the URL and either collect_memstats must be enabled or at least one
+           extra_chart must be defined.
+        """
+
+        if len(self.configuration) < 1:
+            self.error("No JOBs specified in the configuration file, disabling module.")
+            return False
+
+        if len(self.url) == 0:
+            self.error("No URL supplied in configuration, disabling module.")
+            return False
+
+        if len(self.configuration.get('extra_charts', dict())) == 0 and not self.configuration.get('collect_memstats'):
```

Comment:
Fixed in [60ddb7db](https://github.com/firehol/netdata/pull/2280/commits/60ddb7db4f85f76df346ef0af98befa08ac47029)